src/app/components/mnoe-invoices-list/mnoe-invoices-list.coffee

```diff
@@ -0,0 +1,65 @@
+@App.component('mnoeInvoicesList', {
+  templateUrl: 'app/components/mnoe-invoices-list/mnoe-invoices-list.html'
+
+  controller: ($state, MnoeAdminConfig, MnoeInvoices) ->
+    ctrl = this
+
+    ctrl.isPaymentEnabled = MnoeAdminConfig.isPaymentEnabled()
+    ctrl.invoices =
+      list: []
+      nbItems: 10
+      search: {}
+      offset: 0
+      page: 1
+      pageChangedCb: (nbItems, page) ->
+        ctrl.invoices.nbItems = nbItems
+        ctrl.invoices.page = page
+        ctrl.invoices.offset = (page  - 1) * nbItems
+        fetchInvoices(nbItems, ctrl.invoices.offset)
+
+    # Server call
+    ctrl.callServer = (tableState) ->
+      search = updateSearch (tableState.search)
+
+      fetchInvoices(ctrl.invoices.nbItems, ctrl.invoices.offset)
+
+    # Update searching parameters
+    updateSearch = (searchingState = {}) ->
+      search = {}
+      if searchingState.predicateObject
+        for attr, value of searchingState.predicateObject
+          search[ 'where[' + attr + '.like]' ] = value + '%'
+
+      ctrl.invoices.search = search
+      return search
+
+    # The expected date is the 2nd of the month following the invoice period
+    ctrl.calculatePaymentDate = (endOfInvoiceDate) ->
```

Comment:
@ouranos The fact that we invoice the 2nd of each month should be coming from the backend (almost a magic number)

Reply:
For the `ctrl.calculatePaymentDate(endOfInvoiceDate)` function.: please do not duplicate it. This code could be in a service.

Reply:
I agree with the first comment, the second has already been addressed. 
What shall I do about the first one?